rs/nns/governance/src/network_economics.rs

```diff
@@ -361,14 +363,14 @@ impl VotingPowerEconomics {
         }
 
         if let Some(delay) = self.neuron_minimum_dissolve_delay_to_vote_seconds {
-            pub const MIN_DISSOLVE_DELAY_FOR_VOTE_ELIGIBILITY_SECONDS_BOUNDS: RangeInclusive<u64> =
-                (3 * ONE_MONTH_SECONDS)..=(6 * ONE_MONTH_SECONDS);
-
-            if !MIN_DISSOLVE_DELAY_FOR_VOTE_ELIGIBILITY_SECONDS_BOUNDS.contains(&delay) {
+            if !VotingPowerEconomics::NEURON_MIN_DISSOLVE_DELAY_TO_VOTE_SECONDS_BOUNDS
+                .contains(&delay)
+            {
                 let defect = format!(
-                    "neuron_minimum_dissolve_delay_to_vote_seconds ({:?}) must be between three and six months.",
+                    "neuron_minimum_dissolve_delay_to_vote_seconds ({:?}) must be between three \
+                     and six months.",
```

Comment:
It seems easy that this wording could get out of sync with the constant.

Reply:
Will address this later in interests of time